packages/bruno-app/src/components/CollectionSettings/Auth/OAuth1/inputsConfig.js

```diff
@@ -0,0 +1,79 @@
+const inputsConfig = [
```

Comment:
Some of these inputs are required to fill (consumer key/secret), and some will be filled by Bruno during execution if found empty (verifier, access token). Some will only make sense under certain conditions (rsa-private key is only needed when using RSA signing method, but then it's mandatory). 

The UX improvements like hints or auto-disabling controls depending on other choices may be introduced in the future, but I guess it's better left out of this PR.